pkg/asset/manifests/dns.go

```diff
@@ -158,7 +159,7 @@ func (d *DNS) Generate(dependencies asset.Parents) error {
 		config.Spec.PrivateZone = &configv1.DNSZone{
 			ID: zoneID,
 		}
-	case libvirttypes.Name, openstacktypes.Name, baremetaltypes.Name, nonetypes.Name, vspheretypes.Name, ovirttypes.Name:
+	case libvirttypes.Name, openstacktypes.Name, baremetaltypes.Name, nonetypes.Name, vspheretypes.Name, ovirttypes.Name, powervstypes.Name:
```

Comment:
wouldn't this be the same as ibmcloud above?

Reply:
@Karthik-K-N -- can you answer this? i can't tell from looking at your Power VS support pr here https://github.com/openshift/cluster-ingress-operator/pull/658/files but am thinking we may be able to just add the powervs case to the ibmcloud case here (which is what i'm pretty sure Prashanth is suggesting)?

Reply:
Yes, We need to add PowerVS to IBMCloud case block and its tracked here https://issues.redhat.com/browse/MULTIARCH-1603 and also I think as per the slack discussion I see, @miyamotoh already done this.